### PR TITLE
Add docs for Microsoft.AspNetCore.Components namespaces

### PR DESCRIPTION
**Pull request by captainsafia**
Addresses https://github.com/dotnet/aspnetcore/issues/27286.

### CODE REVIEW

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.Circuits.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Components.Server.Circuits">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides classes for implementing a circuit in Blazor Server. Not intended for use outside of the Blazor Server framework.</summary>
```
Comment (pranavkm) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.Circuits.xml:3]: This has types like CircuitHandler which we expect users to author. Maybe 

> Provides classes for implementing and working with circuits in a Blazor Server app.

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Forms.xml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <Namespace Name="Microsoft.AspNetCore.Components.Forms">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides classes for managing form views, state, and validations.</summary>
+    <remarks>
+      For more information, see 
+      <see href="https://docs.microsoft.com/aspnet/core/blazor/forms-validation">
+      ASP.NET Core Blazor forms and validation</see>.
```
Comment (Rick-Anderson) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Forms.xml:7]:
```suggestion
      For more information, see  <see href="https://docs.microsoft.com/aspnet/core/blazor/forms-validation">  ASP.NET Core Blazor forms and validation</see>.
```

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Components.Server">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides classes for configuring circuits and authentication state in Blazor Server applications. Not intended for use outside the Blazor Server framework.</summary>
```
Comment (pranavkm) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.xml:3]: This looks incorrect.

> Provides classes for operating a Blazor Server applications. This namespace includes types such as CircuitOptions that allows fine-grained control of Blazor Server.

**Review by pranavkm — APPROVED** [review]

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Forms.xml (hunk 2 above)
Comment (Rick-Anderson) [reply aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Forms.xml:7]: I don't think there's supposed to be newlines here. @JamesNK  will correct me if I'm wrong.

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Components.xml [hunk 4]
```diff
@@ -1,6 +1,9 @@
 <Namespace Name="Microsoft.AspNetCore.Components">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides classes for component rendering, event handling, and routing.</summary>
+    <remarks>
+      For more information, see <see href="https://docs.microsoft.com/aspnet/core/blazor/components">
+      Create and use ASP.NET Core Razor components</see>.
```
Comment (Rick-Anderson) [inline aspnet-core/xml/ns-Microsoft.AspNetCore.Components.xml:6]:
```suggestion
      For more information, see <see href="https://docs.microsoft.com/aspnet/core/blazor/components"> Create and use ASP.NET Core Razor components</see>.
```

**Review by Rick-Anderson — APPROVED** [review]
Looks great. I really like the following wording

Not intended for use in application code.

Only question is newlines in element.

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Forms.xml (hunk 2 above)
Comment (JamesNK) [reply aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Forms.xml:7]: It won't matter either way. Might as well update for consistency with other xml files.

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.Circuits.xml (hunk 1 above)
Comment (captainsafia) [reply aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.Circuits.xml:3]: Good catch. Will fix.

File: aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.xml (hunk 3 above)
Comment (captainsafia) [reply aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.xml:3]: Hm. Which part is incorrect? I edited the wording here a bit. Let me know what you think.

**Pull request merged**